integreat_compass/cms/constants/offer_version_states.py

```diff
@@ -0,0 +1,15 @@
+"""
+This module contains all string representations of all approval states of offer
+versions, used by :class:`~integreat_compass.cms.models.offers.offer_version.OfferVersion`
+"""
+from django.utils.translation import gettext_lazy as _
+
+REJECTED = 0
+APPROVED = 1
+PENDING = 2
```

Comment:
```suggestion
REJECTED = False
APPROVED = True
PENDING = None
```
In this case, I think boolean values make more sense for these constants...